laravel/Scorpius/resources/views/layouts/templateLogin.blade.php

```diff
@@ -4,17 +4,28 @@
 -->
 
 @include('layouts._includes.top')
-<!-- Aqui ficara as coisas que ficam ao lado esquerdo das telas de login, que no caso, 
-é aquela imagem do background -->
-<div id="lado_esquerdo">
-<img id="img-esquerda" src="" alt="">
-</div>
 
-<!-- Aqui ficara as coisas que ficam ao lado direito das telas de login, como os campos
-os botões, etc.. -->
-<div id="lado_direito">
-<img id="nome-scorpius" src="" alt="">
-@yield('conteudo')
+<div class="row">
```

Comment:
o conflito deve ta aqui. porém é estranho pq o arquivo da master não ta como ta mostrando aqui. era o arquivo como ta nessa branch de layout, mas ele já foi alterado. se o conflito for aqui deixa como ta na master.

Reply:
Pode fazer o que achar melhor, depois eu vejo o que faço.